LICENSE copy.md

```diff
@@ -0,0 +1,9 @@
+All Reports in this Repository are licensed by Contributors
```

Comment:
what is the right term?
* reports
* documents
* ?

Reply:
Reports includes notes and recommendations. This is something I have copied from W3C and would not want to change anything